ssz/sedes/list.py

```diff
@@ -104,5 +110,24 @@ def deserialize_content(self, content: bytes) -> Generator[TDeserialized, None,
         if element_start_index > len(content):
             raise Exception("Invariant: must not consume more data than available")
 
+    #
+    # Tree hashing
+    #
+    def hash_tree_root(self, value: Iterable[TSerializable]) -> bytes:
+        if isinstance(self.element_sedes, BasicSedes):
```

Comment:
Using `hasattr` can buy 1 sec in the benchmark, but I'm super uncertain about this.
```suggestion
        if hasattr(self.element_sedes, "serialize"):
```

Reply:
Nevermind, it's random noise

Reply:
I don't think this is worth it at the moment. And if it turns out that the `isinstance` checks are a major bottleneck maybe we can try to cache it somewhere before reverting to the old `hasattr` checks.